style/js/README.md

```diff
@@ -1,3 +1,163 @@
+JavaScript Style Guides
+=====
+
+Guides for programing JavaScript in style
+
+General
+---
+* Use [ESLint](http://eslint.org/)
+* Always use `use strict`
+
+Parens and Braces
+---
+* Leave one space between `if` and `(`.
+
+```javascript
+if (something) {
+  //some work here;
+}
+```
+
+* Prefer this form for multiline if statements.
+
+```javascript
+if (something) {
+  // do stuff
+  // even more stuff
+} else {
+  // some other stuff
+  // real stuff
+}
+```
+
+* Don't include spaces before `(` and function declaration.
+
+```javascript
+function bar() {
+  return  foo;
+}
+
+var bar = function() {
+  return foo;
+}
+```
+
+Assignments and Declarations
+---
+* Initialize arrays with `[ ]`.
+
+```javascript
+var array = [];
+```
+
+* Initialize objects with `{ }`.
+
+```javascript
+var object = {};
+```
+
+* Leave one space after `{` and before `}` when initializing objects on the same line.
+
+```javascript
+var map = { ready: 9, when: 4, "you are": 15 };
+```
+
+* Prefer single line object/array initialization.
```

Comment:
of course there is a difference (although I disagree that inline should be preferred, and a quick look at the oncostyle code tells me I might not be the only one. But that's not the point here)

An exception is something exceptional (duh). Having large objects (i.e., multiline), it not something exceptional, we do it all the time. So this guideline will only actually apply to a small percentage of all objects

But again, I just found several objects not following this guide in our recent repos. So I have to ask if we actually want this here